middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -1492,7 +1483,8 @@ let simplify_let_cont0 ~(simplify_expr : _ Simplify_common.expr_simplifier) dacc
   let denv_for_join = DE.increment_continuation_scope denv_before_body in
   let denv_for_body = DE.increment_continuation_scope denv_for_join in
   let denv_for_body =
-    DE.add_continuation_defined_in_current_continuation denv_for_body
+    let lifting_cost = Lifted_cont_params.length (DE.variables_defined_in_current_continuation denv_for_body) in
```

Comment:
This has potentially quadratic cost, are we worried enough to cache the length in `Lifted_cont_params`?

Reply:
I added a length cache in `Lifted_cont_params`